lecture10/Instances.yml

```
@@ -0,0 +1,93 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters: 
+  KeyName:
+    Type: AWS::EC2::KeyPair::KeyName
+  MyIP:
+    Type: String
+  RDSMasterUsername:
+    Type: String
+  RDSMasterUserPassword:
+    Type: String
+
+Resources: 
+# ------------------------------------------------------------#
+#  EC2Instance
+# ------------------------------------------------------------#
+  EC2Instance:
+    Type: AWS::EC2::Instance
+    Properties:
+      KeyName: !Ref KeyName
+      NetworkInterfaces:
+        - AssociatePublicIpAddress: true
+          DeviceIndex: 0
+          SubnetId: !ImportValue RaiseTechSampleApp-PublicSubnet1a
+          GroupSet:
+            - !Ref SecGroupForEC2
+      ImageId: ami-04f0680f68f076681
+      InstanceType: t2.micro
+      IamInstanceProfile: !ImportValue RaiseTechSampleApp-InstanceProfile
+      Tags:
+        - Key: Name
+          Value: EC2Instance-CF
+  SecGroupForEC2:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: forEC2-CF
+      VpcId: !ImportValue RaiseTechSampleApp-VPC
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          CidrIp: !Ref MyIP
+          FromPort: 22
+          ToPort: 22
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          SourceSecurityGroupId: !ImportValue RaiseTechSampleApp-SecGroupForALB
+      Tags:
+        - Key: Name
+          Value: SecGroup-EC2-CF
+# ------------------------------------------------------------#
+#  RDSInstance
+# ------------------------------------------------------------#
+  RDSInstance:
+    Type: AWS::RDS::DBInstance
+    Properties:
+      AllocatedStorage: 20 
+      DBInstanceClass: db.t3.micro
+      AvailabilityZone: ap-northeast-1a
+      DBName: forCF
+      DBSubnetGroupName: !ImportValue RaiseTechSampleApp-SubnetGroup
+      Engine: mysql
+      EngineVersion: 8.0.34
+      MasterUsername: !Ref RDSMasterUsername
+      MasterUserPassword: !Ref RDSMasterUserPassword
+      MultiAZ: false
+      Port: 3306
+      PubliclyAccessible: false
+      StorageEncrypted: true
+      StorageType: gp2
+      VPCSecurityGroups:
+        - !Ref SecGroupForRDS
+      Tags:
+        - Key: Name
+          Value: RDSInstance-CF
+  SecGroupForRDS:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: forRDS-CF
+      VpcId: !ImportValue RaiseTechSampleApp-VPC
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 3306
+          ToPort: 3306
+          SourceSecurityGroupId: !Ref SecGroupForEC2
+      Tags:
+        - Key: Name
+          Value: SecGroup-RDS-CF
+
+Outputs:
+  EC2Instance:
+    Value: !Ref EC2Instance
+    Export:
+      Name: RaiseTechSampleApp-EC2

```

パスカルケースの場合、区切り文字にハイフンは使わないので以下のようになるので修正お願いします！
```suggestion
      Name: RaiseTechSampleAppEc2
```

---

OutputsのExport名の`-`を消しました。